File: DeleteKthToLast/DeleteKthToLast.java
import java.io.*;
import java.util.*;
import crio.ds.List.ListNode;

class DeleteKthToLast {
	public static void main(String [] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		ArrayList<Integer> list = new ArrayList<Integer>();
		for ( int i = 0; i < n; i++) {
			int x = sc.nextInt();
			list.add(x);
		}
		int k = sc.nextInt();
		ListNode head = ListNode.createList(list);
		head = Solution.deleteKthToLast(head, k);
		List<Integer> result = ListNode.extractList(head);
		for ( int i = 0; i < result.size(); i++) {

			System.out.print(result.get(i) + " ");
		}
	}
}

File: DeleteKthToLast/DeleteKthToLast.cpp
#include<bits/stdc++.h>
using namespace std;
#include "Solution.cpp"


int main() {
    int n;
    cin >> n;
    vector<int> list(n);
    for(int i = 0 ; i < n ; ++i) {
        cin >> list[i];
    }
    int k;
    cin >> k;

    ListNode* head = ListNode().createList(list);
    ListNode* result = deleteKthToLast(head, k);
    vector<int> resultList = ListNode().extractList(result);
    for(int i = 0 ; i < resultList.size() ; ++i) {
        cout << resultList[i] << " ";
    }
    cout << "\n";
    return 0;
}
